distributedcombigrid/src/sgpp/distributedcombigrid/fullgrid/FullGrid.hpp

```diff
@@ -266,7 +266,8 @@ FullGrid<FG_ELEMENT>::FullGrid(DimType dim, LevelType level, bool hasBdrPoints,
                                const BasisFunctionBasis* basis) {
   // set the basis function for the full grid
   if (basis == NULL)
-    basis_ = LinearBasisFunction::getDefaultBasis();
+    //TODO deal with memory leak
```

Comment:
Maybe switch to smart pointers?

Reply:
Hm, I was considering that (shared_ptr would avoid the ownership ambiguity here). But it would mean wrapping the serialized and sent/received Basis classes into a shared_ptr, and `FullGrid` is rarely ever used in DisCoTec (actually only for tests nowadays). So my suggestion would be to tolerate it for now and to leave the comment in...?